### DISCUSSION TITLE
Reset PocketBase data after each day

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1914] [Q&A]
Hey
I finished building my project and now I would like to showcase it in my resume. The problem is, it contains a real-time chat functionality, so I would like to reset the submitted data by all of the users (other than a specific user (my user account)) every day. How should I implement this functionality, so it automatically does this... I think the PocketBase demo has similar functionality.
Thanks

**Comment by ganigeorgiev** [comment]
There is no built-in functionality for this. It is difficult to generalize and I'm not sure what approach to recommend.

You can implement it on your own if you [use PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) OR setup a cron that will run some shell script.

The shell script could either replace the `pb_data` directory entirely from some backup state, or could execute directly an SQL command using the `sqlite3` cli tool, for example:

```sh
sqlite3 pb_data/data.db 'delete from messages where user != "yourUserId"'
```

_(note that if you decide to replace the entire `pb_data` directory the application must be restarted to pick the new db file)_
